### DISCUSSION TITLE
Can we change the default values  of iou  and conf  for the pre-trained models during the inference?

### DISCUSSION THREAD

**Discussion by halqadasi** [ultralytics/ultralytics#2007] [Q&A]
I have used the pre-trained model below from the PyTorch hub in the inference time. The output of the model here includes the NMS, my question is: **Can we change the default values of ` iou`  and `conf `  which are 0.45 and 0.25, respectively during the inference?**

`model = torch.hub.load('ultralytics/yolov5', 'yolov5s', pretrained=True)`

**Comment by glenn-jocher** [comment]
@halqadasi yes, you can adjust the `iou` and `conf` thresholds during inference with YOLOv8. When you load the model, you can set the `conf` threshold for object confidence and the `iou` threshold for non-maximum suppression directly on the model object. For example, after loading the model, you can modify these attributes like so:

```python
model.conf = 0.30  # Set a new confidence threshold
model.iou = 0.50   # Set a new IoU threshold for NMS
```

Just replace `0.30` and `0.50` with your desired confidence and IoU thresholds, respectively. This will affect how the model filters predictions during the inference process. 🚀

For more detailed information on how to use and configure YOLOv8 models, please refer to our documentation at https://docs.ultralytics.com.
